pkg/network/event_common.go

```diff
@@ -232,10 +232,14 @@ type ConnectionStats struct {
 	Family           ConnectionFamily
 	Direction        ConnectionDirection
 	SPortIsEphemeral EphemeralPortType
-	Tags             uint64
+	StaticTags       uint64
+	Tags             map[string]interface{}
```

Comment:
`map[string]struct{}` is the idiomatic way of doing a set. The Go compiler understands the empty struct and will not use any storage for the value.